core/src/test/scala/org/apache/spark/sql/delta/DeltaConfigSuite.scala

```diff
@@ -158,4 +158,31 @@ class DeltaConfigSuite extends SparkFunSuite
       assert(e.getMessage == msg)
     }
   }
+
+  test("Allow setting Serializable Isolation Level") {
+    // (1) we can set serializable isolation level
+    withTempDir { dir =>
+      sql(
+        s"""CREATE TABLE delta.`${dir.getCanonicalPath}` (id bigint) USING delta
+           |TBLPROPERTIES ('delta.isolationLevel' = 'Serializable')
+           |""".stripMargin)
+
+      val isolationLevel =
+        DeltaLog.forTable(spark, dir.getCanonicalPath).isolationLevel
+
+      assert(isolationLevel == Serializable)
+    }
+
+    // (1) we can not set writable serializable isolation level
+    withTempDir { dir =>
+      val e = intercept[DeltaIllegalArgumentException] {
+        sql(
+          s"""CREATE TABLE delta.`${dir.getCanonicalPath}` (id bigint) USING delta
+             |TBLPROPERTIES ('delta.isolationLevel' = 'WritableSerializable')
```

Comment:
nit: `WriteSerializable`

Reply:
oh geez i was mixing it up with good old Hadoop's Writable. fixed.